lldb/source/Plugins/SymbolLocator/Debuginfod/SymbolLocatorDebuginfod.cpp

```diff
@@ -19,6 +20,43 @@ using namespace lldb_private;
 
 LLDB_PLUGIN_DEFINE(SymbolLocatorDebuginfod)
 
+enum SymbolLookupMode {
+  eLookupModeNever,
+  /*
+  eLookupModeAsync,
+  eLookupModeOnDemand,
+  */
+  eLookupModeAlways,
+};
+
+static constexpr OptionEnumValueElement g_debuginfod_symbol_lookup_mode[] = {
+    {
+        eLookupModeNever,
+        "never",
+        "Do not ever query DEBUGINFOD servers for symbols.",
+    },
+    /*
+    {
+        eLookupModeAsync,
+        "async",
+        "Only query for symbols asynchronously.",
+    },
+    {
+        eLookupModeOnDemand,
+        "on-demand",
+        "Only query DEBUGINFOD servers when they're explicitly requested via "
+        "'target "
+        "symbols add'",
+    },
+    */
+    {
+        eLookupModeAlways,
+        "always",
+        "If symbols aren't available, always check the configured DEBUGINFOD "
+        "servers ",
```

Comment:
Maybe be a bit more descriptive here like:
"Always try to find debug information for any executable or shared library in any debug session as the shared libraries are loaded. Note this can cause a lot of debug information to appear in your project and can slow down your debug session."